src/app/containers/Header/__snapshots__/index.test.jsx.snap

```diff
@@ -83,37 +80,34 @@ exports[`Header should render correctly 1`] = `
     >
       <a
         className="c2 c3"
-        href="https://www.bbc.co.uk/news"
+        href="/undefined"
```

Comment:
Why are we getting undefined here?

Reply:
This was because the source of the url wasn't defined in the test stub. This is now defined